src/Sfa.Tl.ResultsAndCertification.Functions/local.settings.json

```diff
@@ -13,6 +13,7 @@
     "UcasTransferAmendmentsTrigger": "45 21 * 8-10 *",
     "OverallResultCalculationTrigger": "30 20 * * *",
     "IndustryPlacementExtractTrigger": "0 30 8 * * *",
+    "AnalystOverallResultExtractTrigger": "0 18 * * * *",
```

Comment:
Please can we align this with as expected timing.

Reply:
Done